SwiftWisdom/Core/UIKit/UIView+Geometry.swift

```diff
@@ -0,0 +1,229 @@
+//
+//  UIView+Geometry.swift
+//  SwiftWisdom
+//
+//  Created by Logan Wright on 1/19/16.
+//  Copyright © 2016 Intrepid. All rights reserved.
+//
+
+import UIKit
+
+extension UIView {
+    public var ip_width: CGFloat {
+        return CGRectGetWidth(frame)
```

Comment:
bounds is right here, because ip_frameWidth is defined below. However, I'm finding these getters to be a little dated since frame has default getters for say, width and height now